monai/bundle/scripts.py

```diff
@@ -1270,6 +1273,9 @@ def trt_export(
             that the TensorRT tries to fit. The `OPT_BATCH` should be the most frequently used input batchsize in
             the application.
         device: the target GPU index to convert and verify the model.
+        use_onnx: whether to use the ONNX-TensorRT way to export the TensorRT engine-based torchscript model.
+        onnx_input_names: optional input names of the ONNX model.
```

Comment:
Suggest to add a simple description of the default value, it seems "input_0" is used.
Same suggestion on `onnx_output_names `

Reply:
maybe can use the sentence like: `if not given, ...` (see also: https://github.com/Project-MONAI/MONAI/blob/287d058a13ccc434b00640bd2c388337db46afe4/monai/bundle/scripts.py#L1036)